### DISCUSSION TITLE
post/2021-12-23-interview/

### DISCUSSION THREAD

**Discussion by giscus** [zcong1993/blog#48] [Announcements]
# post/2021-12-23-interview/

一点面试感想

https://blog.cong.moe/post/2021-12-23-interview/

**Comment by kbrx93** [comment]
感谢分享✨，持续学习确实非常重要。

**Reply by zcong1993** [reply]
共勉, 另一方面其实能够坚持持续学习的人要么对编程很感兴趣, 要么是自控能力很强的人.

**Comment by genie88** [comment]
学习的深度、语言基础、自我驱动

**Reply by zcong1993** [reply]
深入学习自己感兴趣的东西是效率最高并且是开心的事情, 哈哈

**Comment by StormSpirit22** [comment]
关于 go-zero 缓存设计的文档可以查看这个：https://go-zero.dev/cn/redis-cache.html 
我想博主的意思应该是对于多行的查询不会重复存储数据行记录，而是建立对应的从查询条件到主键的映射，再通过主键 id 来查询缓存里的数据。这样同一条记录在 redis 中只会存储一份完整数据，节省了存储空间。是这个意思吗？

**Reply by zcong1993** [reply]
对的, 就是照着 go-zero 文章和源码实现的.

**Reply by StormSpirit22** [reply]
好的，谢谢博主。
